### PR TITLE
Functional tests - Add some tests in module manager page

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Add some tests in module manager page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Nightly tests => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/4926770800
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/10_seeMore.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Import utils
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+// Import commonTests
+import loginCommon from '@commonTests/BO/loginBO';
+
+// Import pages
+import dashboardPage from '@pages/BO/dashboard';
+import {moduleConfigurationPage} from '@pages/BO/modules/moduleConfiguration';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/10_seeMore.ts]:
```suggestion
```

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/10_seeMore.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import moduleManagerPage from '@pages/BO/modules/moduleManager';
+
+// Import data
+import Modules from '@data/demo/modules';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/10_seeMore.ts]:
```suggestion
```

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    const isModalVisible = await moduleManagerPage.closeUploadModuleModal(page);
+    await expect(isModalVisible).to.be.true;
+  });
+
+  it(`should search the module '${Modules.keycloak.name}'`, async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'searchModule', baseContext);
+
+    const isModuleVisible = await moduleManagerPage.searchModule(page, Modules.keycloak);
+    await expect(isModuleVisible, 'Module is not visible!').to.be.true;
+  });
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts:85]: You should remove the module at the end in POST-COND.

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts:85]: it's removed here => 
```
after(async () => {
    await helper.closeBrowserContext(browserContext);
    await files.deleteFile('module.zip');
  });
```

Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts:85]: Not the zip ;) but the module in PrestaShop.


Not removing, but uninstalling.

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/07_modules/01_moduleManager/08_uploadModule.ts:85]: Done ;) Thanks

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**